doc/try_new_features_without_releasing.md

```diff
@@ -0,0 +1,32 @@
+Try new features without releasing
+===
+
+This documentation shows how to try new features in commit or PR without releasing.
+
+Azure CLI releases new versions every 3 weeks. Many customers and developers are eager to try and test new features before the formal release. Fortunately, Azure CLI project build artifacts for every commit and PR and you can download them.
+
+## Tutorial for commit
+
+Step 1. Select a commit. Click the √ or × symbol. Click "Details" of a "build package" item.
+
+![](assets/1.PNG)
+
+Step 2. Click "View more details on Azure Pipelines".
```

Comment:
Is it able to put a red rectangle around `View more details on Azure Pipelines`? It's quite hard to find it. same with `1 artifact`

Reply:
Good idea!

Reply:
Please input the doc link to OneNote after this PR is merged.

Reply:
> 
> 
> Is it able to put a red rectangle around `View more details on Azure Pipelines`? It's quite hard to find it. same with `1 artifact`

Red rectangle added.

Reply:
> 
> 
> Please input the doc link to OneNote after this PR is merged.

OK.